site/src/pages/components/focusgroup.explainer.mdx

```diff
@@ -221,7 +223,7 @@ The CSS-specified value of 'nowrap' would override the HTML attribute value's "w
 similarly for 'both' overriding "horizontal".
 
 There is no change to the way Tab navigation works with `tabindex` nor the Tab ordering behavior. To
-use a focusgroup, focus must enter that element's focusable children somehow: for accessibility
+use a focusgroup, focus must enter that element's focusable shadow-inclusive children somehow: for accessibility
```

Comment:
The statement of "somehow" feels odd to me given how normative this text reads. Unsure where WHATWG defines this but I recommend adjusting this to possibly something along the lines of "it enters the elements focusable shadow-inclusive children by traversing to the first child in based on standard tab ordering behavior (link out to WHATWG definition/algo)"

Reply:
That was pre-existing bad prose... but since I'm here I'm going to clean it up for the time-being. Note, this paragraph will shortly be obsolete because the feature IS changing to modify how Tab navigation works in certain scenarios. :)